swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md

```diff
@@ -38,13 +38,12 @@ If the output path is not specified, the output will be written to a directory c
  Buffers definition. 
 If the package is not specified, the output will be written to a 'temp' directory in the current location.
 
-`--mode`   - Set the mode (client or service) to generate code samples. If not specified, only the stub file is
- generated.
+`--mode`   - Set the mode as client or service to generate code samples. Set the mode as proxy to generate a gateway proxy. If not specified, only the stub file is generated.
 
 
 ## Sample
 
-The below example shows how you can generate Ballerina source code from the following Protocol Buffers definition (in the `helloworld_service.proto` file).
+The below example shows how you can generate Ballerina source code from the following Protocol Buffer's definition (in the `helloworld_service.proto` file).
```

Comment:
Wouldn't it be much better to use either `Protocol Buffers' definition` or [`Protocol Buffers definition`](https://developers.google.com/protocol-buffers)? 

@praneesha WDYT?

Reply:
Should we say "Protocol Buffer definition" @daneshk?

Reply:
`Protocol Buffers` is the correct name. Shall we keep it as it is?

Reply:
+1 Thanks a lot!